src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
+    public AuthenticationException(String s) {
+    }
```

Comment:
The constructor should call `super(s);` to pass the message to the superclass `RuntimeException`.